src/App.js

```diff
@@ -1,13 +1,67 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList/GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export class App extends React.Component {
+  state = {
+    goods: [],
+  }
 
-export default App;
+  loadAll = (event) => {
+    getAll().then((result) => {
+      this.setState({
+        goods: result,
+      });
+    });
+  };
+
+  get5 = () => {
+    get5First().then((result) => {
+      this.setState({
+        goods: result,
+      });
+    });
+  };
+
+  getRed = () => {
+    getRedGoods().then((result) => {
+      this.setState({
+        goods: result,
+      });
+    });
+  };
```

Comment:
Can you make 1 function out of these 3?
Hint: accept callback function as a parameter (getAll, getRedGoods, get5First)